### PR TITLE
Individual result mod 2.0

### PR DESCRIPTION
**Pull request by mayerm94**
Clause_results and statement_results are an array now, not a hash keyed by statement names

Pull requests into cqm-execution require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

❗️BEFORE MERGE ❗️
- [x] AFTER https://github.com/projecttacoma/cqm-models/pull/113 is merged. Update package.json and Gemfile back to master and update browser.js

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/CYPRESS-514
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name: @hossenlopp 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name: @dczulada
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: lib/models/calculator.js [hunk 1]
```diff
@@ -173,6 +188,11 @@ module.exports = class Calculator {
           if (!resultsByPatient[patientId]) {
             resultsByPatient[patientId] = {};
           }
+
+          if (requestDocument) {
```
Comment (hossenlopp) [inline lib/models/calculator.js]: If just sending back a 'document' formatted object, then perhaps just do nothing instead of mongoosing to just toObject it as it leaves the service.

File: lib/helpers/results_helpers.js [hunk 2]
```diff
@@ -205,84 +209,117 @@ module.exports = class ResultsHelpers {
...
               localId,
               clause,
               rawResult: clauseResult.raw,
             });
-            clauseResults[libraryName][localId] = clauseResult;
+            if (requestDocument) {
+              const clauseResultDocument = new CqmModels.ClauseResult(clauseResult);
+              clauseResults.push(clauseResultDocument);
+            } else {
+              clauseResults[library_name][localId] = clauseResult;
```
Comment (hossenlopp) [inline lib/helpers/results_helpers.js]: This is not getting hit. I think you need a test that does both `includeClauseResults = true` and `requestDocument = false`.
